src/treegen.cpp

```diff
@@ -183,6 +197,17 @@ treegen::error make_ltree(MMVManip &vmanip, v3s16 p0,
 	std::stack <core::matrix4> stack_orientation;
 	std::stack <v3f> stack_position;
 
+	// fill NodeDefs with unique id if requested
+	if (tree_definition.enable_unique_ids) {
+		u8 unique_id = 0;
+		PcgRandom r((((p0.X << 16) + p0.Y) << 16) + p0.Z);
+		unique_id = r.next();
+		if (!unique_id) unique_id = 1;
```

Comment:
Why have the condition body on the same line as the condition?  Surely, you don't have anything to hide here, right? ;)